.github/workflows/haskell.yml

```diff
@@ -103,8 +105,19 @@ jobs:
     - name: Build
       run: cabal build all
 
+    - name: Set test timeout (Unix)
+      if: ${{ startsWith(matrix.os, 'ubuntu') || startsWith(matrix.os, 'macOS') }}
+      run: |
+        export TASTY_TIMEOUT="5m"
+
+    - name: Set test timeout (Windows)
+      if: ${{ startsWith(matrix.os, 'windows') }}
+      run: |
+        $env:TASTY_TIMEOUT="5m"
```

Comment:
these are not preserved.

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable